Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -105,7 +105,7 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
                 actions.append(threadReplyActionItem())
             }
 
-            if channelConfig.readEventsEnabled {
+            if channelConfig.readEventsEnabled && (!message.isPartOfThread || message.showReplyInChannel) {
```

Comment:
There are missing tests here 🤔 We already have tests for `test_messageActions_`, we should add these ones as well ideally

Reply:
Added